packages/kbn-search-api-panels/components/code_box.tsx

```diff
@@ -81,7 +83,8 @@ export const CodeBox: React.FC<CodeBoxProps> = ({
     <EuiThemeProvider colorMode="dark">
       <EuiButtonEmpty
         aria-label={i18n.translate('searchApiPanels.welcomeBanner.codeBox.selectAriaLabel', {
-          defaultMessage: 'Select a programming language',
+          defaultMessage: 'Select a programming language for the {context} code snippet',
```

Comment:
@rshen91, how will it look in other places where we didn't pass `consoleTitle`?

<img width="1082" alt="image" src="https://github.com/user-attachments/assets/600bac17-b510-4d78-990a-eac6d0ee12aa">

I suspect it will be something like `Select a programming language for the **undefined** code snippet.`

Reply:
It just leaves it blank 😌 ! So it looks like Select a programming language for the  code snippet

Reply:
Yep, it sounds normal in English. But I'm not sure that the same will happen with other languages ​​that we support.

Reply:
I just mean maybe we define 2 translations?

Reply:
ah ok I think I am following - [f63a6e0](https://github.com/elastic/kibana/pull/188363/commits/f63a6e03304950ee585e3bd221fb25a8114b95b8) has my changes thank you!